Source/HelixToolkit.SharpDX.Core.Tests/HelixToolkit.SharpDX.Core.Tests.csproj

```diff
@@ -7,9 +7,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
-        <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
-        <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
```

Comment:
Since there was only one test unit MS Test I switched that to NUnit before adding more tests.